azure-storage-js-v10-quickstart-v3/README.md

```diff
@@ -0,0 +1,62 @@
+---
+page_type: sample
+name: "Using JavaScript SDK v10 for Azure Storage"
+description: "Quickstart sample for the Azure Storage v10 SDK for JavaScript."
+languages:
+- javascript
+- nodejs
+products:
+- azure
+- azure-storage
+---
+
+# Quickstart - JavaScript SDK v10 for Azure Storage
+
+This repository implements the [quickstart](http://docs.microsoft.com/azure/storage/blobs/storage-quickstart-blobs-nodejs-v10) sample for the [Azure Storage v10 SDK for JavaScript](https://github.com/Azure/azure-storage-js).
+
+## Set up
+First, clone the repository on your machine:
+
+```bash
+git clone https://github.com/Azure-Samples/azure-storage-js-v10-quickstart.git 
+```
+
+Then, switch to the appropriate folder:
+
+```bash
+cd azure-storage-js-v10-quickstart-v3
+```
+
+Next, install the dependencies:
+
+    npm install
+
+Finally, rename the file `.env.example` to `.env` and add your values for *AZURE_STORAGE_ACCOUNT_NAME* and *AZURE_STORAGE_ACCOUNT_ACCESS_KEY*.
```

Comment:
Please tell the user where to get this data.  Also, if there are pre-reqs, make sure they are clearly called out at the top of this file.